lib/order_reporting.rb

```diff
@@ -1,12 +1,24 @@
 require 'order_reporting/engine'
 require 'order_reporting/report'
+require 'order_reporting/schedulers/delayed_job_scheduler'
+require 'order_reporting/queries/latest_orders_query'
```

Comment:
This is an example query used within the tests. It gets completed orders from last 24 hours. Figured it was a sensible default to provide with the library.